### ISSUE TITLE
Bump minimum API version

### ISSUE THREAD

**Issue by antonlukin** [stevebest/passport-vkontakte#52]
According new [API Version Life Cycle](https://vk.com/dev/constant_version_updates) we should bump minimum API version to prevent current error:

```json
{"success":false,"message":"Invalid request: versions below 5.21 are deprecated. Version param should be passed as \"v\". \"version\" param is invalid and not supported. For more information go to https://vk.com/dev/constant_version_updates"}
```

**Comment by AndrewTkachenko** [comment]
@antonlukin for fix it in production use `apiVersion` option for set valid vk api version. 
Note: Valid version is 5.89, not 5.21 like in error response from API

**Comment by antonlukin** [comment]
@AndrewTkachenko I know about it, but suggest to add minimum required api version by default

**Comment by stevebest** [comment]
Anyone willing to submit a PR?

**Comment by antonlukin** [comment]
I'll do it soon

**Comment by stevebest** [comment]
Merged #53 and published as 0.3.3

Could you please confirm it is working as intended? (I have deleted my own VK account long time ago and have no intention to start a new one, even for testing.)

Thanks!

**Comment by antonlukin** [comment]
Looks working. Thanks!
